{"timestamp":"2025-10-05T10:31:36.299682Z","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager","filename":"manager.py","lineno":179}
{"timestamp":"2025-10-05T10:31:36.301788Z","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/retail_transaction.py","logger":"airflow.models.dagbag.DagBag","filename":"dagbag.py","lineno":593}
{"timestamp":"2025-10-05T10:31:36.324463Z","level":"warning","event":"The `airflow.operators.python.PythonOperator` attribute is deprecated. Please use `'airflow.providers.standard.operators.python.PythonOperator'`.","category":"DeprecatedImportWarning","filename":"/opt/airflow/dags/retail_transaction.py","lineno":5,"logger":"py.warnings"}
{"timestamp":"2025-10-05T10:31:36.448823Z","level":"info","event":"Task instance is in running state","logger":"task.stdout"}
{"timestamp":"2025-10-05T10:31:36.450502Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","logger":"task.stdout"}
{"timestamp":"2025-10-05T10:31:36.452016Z","level":"info","event":"Current task name:load_task","logger":"task.stdout"}
{"timestamp":"2025-10-05T10:31:36.452887Z","level":"info","event":"Dag name:etl_retail_transactions_parquet","logger":"task.stdout"}
{"timestamp":"2025-10-05T10:31:36.535586Z","level":"error","event":"Task failed with exception","logger":"task","filename":"task_runner.py","lineno":994,"error_detail":[{"exc_type":"TypeError","exc_value":"'_ConnectionFairy' object does not support the context manager protocol","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1307,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":416,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":216,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":239,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":82,"name":"run"},{"filename":"/opt/airflow/dags/retail_transaction.py","lineno":149,"name":"load"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-10-05T10:31:36.557330Z","level":"info","event":"Task instance in failure state","logger":"task.stdout"}
{"timestamp":"2025-10-05T10:31:36.558620Z","level":"info","event":"Task start","logger":"task.stdout"}
{"timestamp":"2025-10-05T10:31:36.559616Z","level":"info","event":"Task:<Task(PythonOperator): load_task>","logger":"task.stdout"}
{"timestamp":"2025-10-05T10:31:36.560509Z","level":"info","event":"Failure caused by '_ConnectionFairy' object does not support the context manager protocol","logger":"task.stdout"}
